### PR TITLE
[ocf] Device name and BLE address set on command line

### PR DESCRIPTION
**Pull request by jprestwo**
Signed-off-by: James Prestwood <james.prestwood@intel.com>

### CODE REVIEW

File: src/zjs_ocf_ble.c [hunk 1]
```diff
@@ -81,6 +84,20 @@ static jerry_value_t ocf_set_ble_address(const jerry_value_t function_val,
...
 {
+#ifdef ZJS_CONFIG_BLE_ADDRESS
+    static const struct bt_storage storage = {
+            .read = zjs_ble_storage_read,
+            .write = NULL,
+            .clear = NULL,
+    };
+
+    if (str2bt_addr_le(default_ble, "random", &id_addr) < 0) {
+        return zjs_error("bad BLE address string");
```
Comment (jimmy-huang) [inline src/zjs_ocf_ble.c]: why is this returning zjs_error() when the function is void()?

File: Makefile [hunk 2]
```diff
@@ -43,7 +43,8 @@ endif
 OCF_ROOT ?= deps/iotivity-constrained
 JS ?= samples/HelloWorld.js
 VARIANT ?= release
-BLE_NAME ?= "Zephyr OCF Node"
+DEVICE_NAME ?= "Zephyr OCF Node"
+BLE_ADDR ?= "none"
```
Comment (jimmy-huang) [inline Makefile]: should this be labeled "random" instead of none?

File: src/zjs_ocf_common.c [hunk 3]
```diff
@@ -205,7 +205,7 @@ static void issue_requests(void)
 static void app_init(void)
 {
     oc_init_platform("ZJS", NULL, NULL);
-    oc_add_device("/oic/d", "oic.d.zjs", "ZJS Device", "1.0", "1.0", NULL, NULL);
+    oc_add_device("/oic/d", "oic.d.zjs", CONFIG_BLUETOOTH_DEVICE_NAME, "1.0", "1.0", NULL, NULL);
```
Comment (jimmy-huang) [inline src/zjs_ocf_common.c]: Would this still work if the user doesn't set the bluetooth name?

File: Makefile (hunk 2 above)
Comment (jprestwo) [inline Makefile]: "none" doesnt get into the C code, its just for the Makefile.ocf_zephyr to check. If it finds "none" it wont define the ZJS_CONFIG_BLE_ADDRESS

File: src/zjs_ocf_common.c (hunk 3 above)
Comment (jprestwo) [inline src/zjs_ocf_common.c]: Yes, its a conditional set in the main Makefile. "Zephyr OCF node"

**Comment by grgustaf** [comment]
+1

**Pull request merged**